ste/jobStatusManager.go

```diff
@@ -67,10 +67,7 @@ func (jm *jobMgr) SendJobPartCreatedMsg(msg JobPartCreatedMsg) {
 		}
 	}()
 	if jm.jstm.partCreated != nil { // Sends not allowed if channel is closed
-		select {
-		case jm.jstm.partCreated <- msg:
-		case <-jm.jstm.statusMgrDone: // Nobody is listening anymore, let's back off.
-		}
+		jm.jstm.partCreated <- msg
```

Comment:
Is this change part of this PR?

Reply:
It shouldn't be, I think I had some issues with the bases of my branch